### DISCUSSION TITLE
How to remove the [Docker]in the prompt

### DISCUSSION THREAD

**Discussion by clifinger** [starship/starship#6414] [Q&A] [answered]
Hello guys,

I want to remove the [Docker]
![Screenshot 2024-11-26 085348](https://github.com/user-attachments/assets/718b0d97-b863-4694-b476-4a3b05350fe0)

**Comment by florian-ariasu** [comment, marked as answer]
Hello! Go to your starship config file starship.toml or whatever you called it and delete these lines in the file: 
[docker_context]
symbol = " " between double quotes should be the docker whale icon but it cannot be rendered here. Please tell me if it worked, if it did not please remove the variable corresponding to docker in the format string that defines how your prompt renders as well.

**Comment by davidkna** [comment]
This looks like the `container` module, you can confirm this by inspecting the output of `stasrhip explain`. The module can be disabled via `starship config container.disabled true`.

**Comment by clifinger** [comment]
Thank you guys, it work :)
